utility/src/main/java/org/oppia/android/util/platformparameter/PlatformParameterValue.kt

```diff
@@ -1,20 +1,22 @@
 package org.oppia.android.util.platformparameter
 
+import org.oppia.android.app.model.PlatformParameter
```

Comment:
Was it giving build errors before? I don't see it used in this file but maybe it's an indirect dependency.

Reply:
This is here because we have mentioned this class in the KDoc.